modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -3,27 +3,34 @@
 // * osd_planning/aws-ccs.adoc
 // * rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.adoc
 // * rosa_planning/rosa-sts-aws-prereqs.adoc
-// * rosa_planning/rosa-hcp-prereqs.adoc
 
 ifeval::["{context}" == "rosa-sts-aws-prereqs"]
 :fedramp:
 endif::[]
-ifeval::["{context}" == "rosa-hcp-aws-prereqs"]
-:fedramp:
-endif::[]
 
 :_mod-docs-content-type: PROCEDURE
+// Conditionals are to change the title (and preserve anchor IDs) when displayed on the rosa-sts-aws-prereqs page
+ifeval::["{context}" == "rosa-sts-aws-prereqs"]
+[id="rosa-classic-firewall-prerequisites_{context}"]
+= ROSA Classic
+endif::[]
+ifeval::["{context}" != "rosa-sts-aws-prereqs"]
 [id="osd-aws-privatelink-firewall-prerequisites_{context}"]
 = AWS firewall prerequisites
+endif::[]
 
-ifdef::openshift-rosa[]
 [IMPORTANT]
 ====
-Only ROSA clusters deployed with PrivateLink can use a firewall to control egress traffic.
+Only clusters deployed with PrivateLink can use a firewall to control egress traffic.
 ====
-endif::[]
 
+ifeval::["{context}" != "rosa-sts-aws-prereqs"]
 This section provides the necessary details that enable you to control egress traffic from your {product-title} cluster. If you are using a firewall to control egress traffic, you must configure your firewall to grant access to the domain and port combinations below. {product-title} requires this access to provide a fully managed OpenShift service.
+endif::[]
+
+.Requirements
```

Comment:
s/.Requirements/.Prerequisites

Per the documentation guidelines these need to be prerequisites.